models/u2f.js

```diff
@@ -0,0 +1,297 @@
+'use strict';
+
+const apiKey = require('../models/api-key.js');
+const encryption = require('../helpers/encryption.js');
+const response = require('../helpers/response.js');
+const u2f = require('u2f');
+const uuid = require('uuid');
+
+module.exports.createAuthentication = (apiKeyValue, apiSecret, u2fUuid, callback) => {
+  console.log('Starting create U2F authentication for uuid: ' + u2fUuid);
+  apiKey.getActivatedApiKey(apiKeyValue, apiSecret, (error, apiKeyRecord) => {
+    if (error) {
+      console.log('Unable to get activated API Key in order to create U2F:', error);
+      response.returnError(401, 'Unauthorized', callback);
+      return;
+    }
+
+    if (!u2fUuid) {
+      console.log('U2F delete request had no UUID.');
```

Comment:
Is "delete" right here?

Reply:
Good catch. I'll fix that.

Reply:
This is kind of strange. This comment used to be on a different line (19), where "delete" wasn't the right word. Now it's on line 137, where "delete" is the right word. Oh well, the code is now correct in that regard.